recipes/libbasisu/all/patches/cmakelist_build_lib.patch

```diff
@@ -0,0 +1,161 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,14 +1,14 @@
+-project(basisu)
+-
+ cmake_minimum_required(VERSION 3.0)
+-option(BUILD_X64 "build 64-bit" TRUE)
+-option(STATIC "static linking" FALSE)
+-option(SSE "SSE 4.1 support" FALSE)
++
++project(libbasisu VERSION 1.15.0 LANGUAGES CXX)
```

Comment:
Not a good idea to change project name, then rely on it for library target.

Then you ends up with ugly liblib file name:

```
libbasisu/1.15.0 package(): Packaged 1 '.a' file: liblibbasisu.a
```